PsfLauncher/main.cpp

```diff
@@ -304,7 +315,87 @@ void LaunchMonitorInBackground(std::filesystem::path packageRoot, const wchar_t
         }
     }
 }
+
+void LogTelemetryData()
+{
+	auto configRoot = PSFQueryConfigRoot();
+
+	if (auto applications = configRoot->as_object().try_get("applications"))
+	{
+		for (auto& applicationsConfig : applications->as_array())
+		{
+			auto exeStr = applicationsConfig.as_object().try_get("executable")->as_string().wide();
+			auto idStr = applicationsConfig.as_object().try_get("id")->as_string().wide();
+			TraceLoggingWrite(
+				g_Log_ETW_ComponentProvider,
+				"ApplicationsConfigdata",
+				TraceLoggingWideString(exeStr, "applications_executable"),
+				TraceLoggingWideString(idStr, "applications_id"),
+				TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
+				TraceLoggingKeyword(MICROSOFT_KEYWORD_CRITICAL_DATA));
```

Comment:
Should this be a measure of critical data?  Critical data would require extra scrutiny and approvals.

Reply:
Yes, we should leave it to critical because the Msix app using PSF usage is very low from current msix report. Infact for the month of April, I see only one app using PSF and the launch instances is pretty low. If we sample the data, it might be hard to make meaningful insights.